### PR TITLE
Fetch API and comments from Kira Miller

### PR DESCRIPTION
**Pull request by VadimDmitr**
Fetch API and comments from Kira Miller

### CODE REVIEW

**Comment by kiramiller425** [comment]
Hi Vadim,

I looked at your lesson 6.2. It looks great. Good job doing the stretch goal as well!

One thing: you do need to focus on more consistent spacing and indentation. Here is a quick way to automatically format your code: https://beautifier.io/
Go to that site in your browser, cut and paste in your JS code, click on the big green "Beautify Code" button on that site, then click on their "Copy to Clipboard" button, then paste into your own JS file.
It will give you a copy of your JS code with consistent spacing and indentation.
You can use this site for your HTML and CSS as well.
You can leave the default settings, or you can adjust the settings. This is a quick, easy way to ensure your code follows whatever standard your manager wants.
There are other programs which will beautify code, as well.

If you want easier control over your layout, try setting the values on lines 6 and 7 to 0px in your index.css. Then you can make simple changes to the nav and other sections as you see fit. You can even use flexbox for your top nav, which will give you less unexpected behavior than float: left;  Float was the old way to control layouts before flexbox. Float really has its quirks that are hard to control.

For creating a public portfolio, you can kind of follow these instructions, but ignore the themes part, as you won't do that: https://medium.com/@brianmwevi/8-steps-to-publish-your-portfolio-on-github-9d6e6e3d2e84
Basically, do step 1. Then git clone that new repo to your local computer. Then copy files from your project into that folder. Then do the instruction's steps 7 and 8.

Keep up the great work!
-Kira Miller
